src/components/Sections/SectionActions.tsx

```diff
@@ -0,0 +1,35 @@
+import React from 'react'
+
+import ActionsMenuItem from 'cozy-ui/transpiled/react/ActionsMenu/ActionsMenuItem'
+import FileIcon from 'cozy-ui/transpiled/react/Icons/File'
+import ListItemIcon from 'cozy-ui/transpiled/react/ListItemIcon'
+import ListItemText from 'cozy-ui/transpiled/react/ListItemText'
+import { makeActions } from 'cozy-ui/transpiled/react/ActionsMenu/Actions'
+import { Action, DisplayMode } from 'components/Sections/SectionsTypes'
+import Icon from 'cozy-ui/transpiled/react/Icon'
+
+const createCustomAction = (primaryText: DisplayMode): (() => Action) => {
```

Comment:
maybe this name can be more specific like `createSectionAction` and adapt all the wording around this new name